app/javascript/dashboard/components/ChannelSelector.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="small-6 medium-4 large-3 channel" @click="$emit('click')">
+    <img :src="src" />
+    <h3 class="channel__title">
+      {{ title }}
+    </h3>
+  </div>
```

Comment:
```suggestion
  <button class="small-6 medium-4 large-3 channel" @click="$emit('click')">
    <img :src="src" />
    <h3 class="channel__title">
      {{ title }}
    </h3>
  </button>
```

Should be a button if it's clickable

Reply:
Done.